### PR TITLE
Drop support for PHP < 7 and upgrade phpunit to version 6

### PR DESCRIPTION
**Pull request by milhomem**
Just to keep version up-to-date and enjoy new features.

### CODE REVIEW

**Comment by milhomem** [comment]
@cloudson what do you think ?

File: composer.json [hunk 1]
```diff
@@ -16,12 +16,12 @@
         }
     },
     "require": {
-        "php": ">=5.5.0",
+        "php": ">=7.0",
```
Comment (cloudson) [inline composer.json]: I agree to use 7.0 and 7.1 on Travis but I'm not comfortable to exclude the 5.6.* yet. 
The version [5.6.30](http://php.net/ChangeLog-5.php#5.6.30) was released two months ago, maybe there are people under that version.

Comment (milhomem) [inline composer.json]: Last release was done more than one year now, we have a such stable product :). Others can still using release v1.0.0 though.

**Comment by cloudson** [comment]
Done! thank you again @milhomem

**Pull request merged**